plugins/action_modify_record/src/modifyRecord.ts

```diff
@@ -2,16 +2,8 @@ import { ActionQueueStatus } from '../../../src/generated/graphql'
 import { ActionPluginType } from '../../types'
 import databaseMethods, { DatabaseMethodsType } from './databaseMethods'
 import { DBConnectType } from '../../../src/components/databaseConnect'
-import { mapValues, get, snakeCase } from 'lodash'
-import { objectKeysToSnakeCase } from '../../../src/components/utilityFunctions'
-import { singular } from 'pluralize'
-
-// This will be prepended to NEW table created if not already present
-export const DATA_TABLE_PREFIX = 'data_table_'
-
-// These are the only tables in the system that we allow to be mutated with this
-// plugin. All other names will have "data_table_" prepended.
-const ALLOWED_TABLE_NAMES = ['user', 'organisation', 'application', 'file']
```

Comment:
Since these can be shared elsewhere, I've moved the `allowedTableNames` list into `config.ts`